Packs/Box/Integrations/BoxV2/BoxV2.py

```diff
@@ -622,6 +622,8 @@ def chunk_upload(self, file_name: Optional[str], file_size: int, file_path: str,
                                                    the request made to commit the file once it has
                                                    been uploaded.
         """
+        self._base_url = 'https://upload.box.com/api/2.0'
+        #  Because of _course_ they have a separate base_url for uploads
```

Comment:
[nO THis totALly mAkEs sEnse AND Is INtUitIvE](https://www.youtube.com/watch?v=aS0-P4JR9PU)